site/static/js/feedback.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+// Init GTM dataLayer for Analytics
+let eventValue = 0;
+let rating = '';
+
+$('body').on('click', '.c-thermometer__emojis a', function () {
+    $(this).addClass('selected');
+});
+
+$('body').on('click', '.c-thermometer-modal__footer .btn', function () {
+    if ($(this).hasClass('btn-link')) {
+        $(this).parents('.c-thermometer-modal__container').fadeOut();
+        $('a.rate-this-page-action').removeClass('selected');
+        $('.c-thermometer-modal__container textarea').val('');
+    } else {
+        const currentString = $(".c-thermometer-modal__container textarea").val()
+
+        if (eventValue > 0) {
+            // Submit DataLayer Event
+            const dataLayer = window.dataLayer || [];
+            dataLayer.push({
+                event: 'rateThisPage',
```

Comment:
Thanks All! Just a heads up that this is on my radar and will hopefully loop back by early next week. 
Also looping in @nat-gunner (Kevin) for FYI and re: GTM naming conventions and object structure.

Reply:
thanks for the heads up @emdecr !
The object structure looks good to me. when that event gets pushed to the datalayer, I can trigger a signal to Google Analytics and track the submission there. 
@cwarnermm you'll be able to see a GA report of all the successful submissions and the page they occurred on, along with a count of the ratings. 
(I won't store the feedback itself, but I'm guessing there's a platform where you can see all that.)

Reply:
Thanks, @nat-gunner! Really appreciate your help with this! @sri-byte's team will likely follow the current docs feedback model to surface user feedback in Looker and a Mattermost community channel.